src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -1141,6 +1133,7 @@ namespace Akka.Actor
         public override Akka.Actor.ICell Underlying { get; }
         public override Akka.Actor.IActorRef GetChild(System.Collections.Generic.IEnumerable<string> name) { }
         public override Akka.Actor.IInternalActorRef GetSingleChild(string name) { }
+        protected virtual Akka.Actor.ActorCell NewActorCell(Akka.Actor.Internal.ActorSystemImpl system, Akka.Actor.IInternalActorRef self, Akka.Actor.Props props, Akka.Dispatch.MessageDispatcher dispatcher, Akka.Actor.IInternalActorRef supervisor) { }
```

Comment:
overrideable method for producing an `ActorCell` - this replaces the delegates we were using in the now-removed `protected internal` LocalActorRef constructor.